web/src/main/java/org/cbioportal/web/parameter/StudyViewFilter.java

```diff
@@ -156,5 +157,22 @@ public void setGenericAssayDataFilters(List<GenericAssayDataFilter> genericAssay
     public void setCustomDataFilters(List<ClinicalDataFilter> customDataFilters) {
         this.customDataFilters = customDataFilters;
     }
+    
+    @JsonIgnore
+    public boolean isSingleStudyUnfiltered() {
```

Comment:
We should probably move this into `StudyViewFilterUtil` class. Let's avoid putting business logic inside model classes.

Reply:
Also, don't we want this cache enabled for multiple studies query